design-library/src/components/BccGraphic/BccGraphic.vue

```diff
@@ -0,0 +1,26 @@
+<script setup lang="ts">
+import { OpenInNewIcon, CheckCircleFillIcon } from "@bcc-code/icons-vue";
+import BccPoster from "./BccPoster.vue";
+
+type Props = {
+  bannerSrc?: string;
+  logoSrc?: string;
```

Comment:
Is it correct that these props are optional? What should the component render if no props are given? Perhaps add a story for that.